scripts/release/build/check-package-dependencies.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const chalk = require('chalk');
+const {readJson} = require('fs-extra');
+const {join} = require('path');
+const {dependencies, projects} = require('../config');
+const {logPromise} = require('../utils');
+
+const check = async ({cwd}) => {
+  const rootPackage = await readJson(join(cwd, 'package.json'));
+
+  const projectPackages = [];
+  for (let i = 0; i < projects.length; i++) {
+    const project = projects[i];
+    projectPackages.push(
+      await readJson(join(cwd, join('packages', project), 'package.json'))
+    );
+  }
+
+  const invalidDependencies = [];
+
+  const checkModule = async module => {
```

Comment:
I probably don’t know async/await well enough, but why does this function need to be marked `async`? It looks fully synchronous internally.

Reply:
Yeah, probably just a difference between how I thought I'd write the code and how I ended up writing it. You're right that `async` could be removed from this function.